core/controllers/reader.py

```diff
@@ -137,18 +104,13 @@ def get(self, exploration_id):
         if (feconf.EXPLORATION_URL_EMBED_PREFIX in self.request.uri or
                 self.request.get('iframed')):
             self.values['iframed'] = True
```

Comment:
Is this value actually used?

If not, what actually handles the "iframed" parameter?